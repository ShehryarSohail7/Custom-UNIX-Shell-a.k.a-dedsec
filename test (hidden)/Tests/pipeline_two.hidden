cat names.txt | tr [:lower:] [:upper:]
cat names.txt | tr [:lower:] [:upper:] | cat -E
echo "Welcome to {CS 370}" | tr "{}" "()"
echo "My password is {12345678}" | tr "{}" "()" | tr -d [:digit:] | tr [:lower:] [:upper:]
find . -type d | grep Tests