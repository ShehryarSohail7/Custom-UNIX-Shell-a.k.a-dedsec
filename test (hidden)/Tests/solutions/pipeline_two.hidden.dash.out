SOAHIB KH
ABDULLAH SOMUURO
ABDUL RIFE
ABDULLAH RROLUSOAHIB KH$
ABDULLAH SOMUURO$
ABDUL RIFE$
ABDULLAH RROLUWelcome to (CS 370)
MY PASSWORD IS ()
./Tests
./Tests/test_output
